examples/src/models/chat/integration_openai.ts

```diff
@@ -71,3 +71,29 @@ AIMessage {
   }
 }
 */
+
+// Coerce response type with JSON mode.
+// Requires "gpt-4-1106-preview" or later
+const jsonModeModel = new ChatOpenAI({
+  modelName: "gpt-4-1106-preview",
+  maxTokens: 128,
+}).bind({
+  response_format: {
+    type: "json",
+  },
+});
+
+// Must be invoked with a system message containing the string "JSON":
+// https://platform.openai.com/docs/guides/text-generation/json-mode
+const res = await jsonModeModel.invoke([
+  ["system", "Only return JSON"],
+  ["human", "Hi there!"],
+]);
+console.log(res);
+
+/*
+  AIMessage {
+    content: '```json\n{\n  "response": "Hello! How can I assist you today?"\n}\n```',
+    additional_kwargs: { function_call: undefined }
+  }
```

Comment:
This doesn't look right with the backticks. I'm guessing because the `response_format` was not properly set?

Reply:
That explains a lot lol